clang/test/Driver/sycl-offload-new-driver.c

```diff
@@ -188,7 +188,7 @@
 // RUN: %clangxx -fsycl -### -fsycl-targets=spir64_gen --offload-new-driver \
 // RUN:   -Xsycl-target-backend=spir64_gen "-device pvc,bdw" %s 2>&1 \
 // RUN:   | FileCheck -check-prefix COMMA_FILE %s
-// COMMA_FILE: llvm-offload-binary{{.*}} "--image=file={{.*}}pvc@bdw{{.*}},triple=spir64_gen-unknown-unknown,arch=pvc,bdw,kind=sycl,compile-opts=-device_options pvc -ze-intel-enable-auto-large-GRF-mode"
+// COMMA_FILE: llvm-offload-binary{{.*}} "--image=file={{.*}}pvc@bdw{{.*}},triple=spir64_gen-unknown-unknown,arch=pvc,arch=bdw,kind=sycl,compile-opts=-device_options pvc -ze-intel-enable-auto-large-GRF-mode -device pvc,compile-opts=bdw"
```

Comment:
Not sure I understand, how to read this command line. These are key-value pairs:
```
file={{.*}}pvc@bdw{{.*}}
triple=spir64_gen-unknown-unknown
arch=pvc
arch=bdw
kind=sycl
compile-opts=-device_options pvc -ze-intel-enable-auto-large-GRF-mode -device pvc
compile-opts=bdw
```
Specifically, how do we decide which compile-opts correspond to which arch. Just in order of appearance?
Also what is `compile-opts=bdw` - is it a valid option for some tool? I mean `bdw`?